apps/OboeTester/app/src/main/res/layout/activity_main.xml

```diff
@@ -182,25 +182,13 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toEndOf="@+id/textView" />
 
-
-    <CheckBox
-        android:id="@+id/setSpeakerphoneOn"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="6dp"
-        android:checked="false"
-        android:onClick="onSetSpeakerphoneOn"
-        android:text="setSpeakerphoneOn/Off()"
-        app:layout_constraintStart_toStartOf="@+id/useCallback"
-        app:layout_constraintTop_toBottomOf="@+id/useCallback" />
-
     <LinearLayout
         android:id="@+id/layoutBluetoothSco"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:orientation="horizontal"
-        app:layout_constraintStart_toStartOf="@+id/setSpeakerphoneOn"
-        app:layout_constraintTop_toBottomOf="@+id/setSpeakerphoneOn"
+        app:layout_constraintStart_toStartOf="@+id/callbackSize"
+        app:layout_constraintTop_toBottomOf="@+id/callbackSize"
         >
         <CheckBox
             android:id="@+id/setBluetoothScoOn"
```

Comment:
Should we move setBluetoothScoOn into CommunicationDeviceView too? setCommunicationDevice is meant to replace it as well.

Reply:
Yes, it was made to deprecate startBluetoothSco and stopBluetoothSco

Reply:
This doc mentions startBluetoothSco() but does not mention whether it replaces setBluetoothScoOn().
https://developer.android.com/reference/android/media/AudioManager#setCommunicationDevice(android.media.AudioDeviceInfo)

Reply:
#1671 to track remaining SCO work